invision-app/app/components/movieListFallback/movieListFallback.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
```

Comment:
Please, use the same structure with **index** files for all components

Reply:
done